src/App.tsx

```diff
@@ -1,24 +1,75 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState, useRef, useMemo } from 'react';
 import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import { getTodos, USER_ID } from './api/todos';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList';
+import classNames from 'classnames';
+
+enum FilterType {
+  all = 'all',
+  active = 'active',
+  completed = 'completed',
+}
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [errorLoad, setErrorLoad] = useState(false);
```

Comment:
u can create `enum` for all the error (`ErrorType.Default = ''`)
But up to u
```suggestion
  const [errorLoad, setErrorLoad] = useState<ErrorType>(ErrorType.Default);
```